koku/masu/util/aws/aws_post_processor.py

```diff
@@ -1,19 +1,20 @@
 import json
 import logging
 
-import ciso8601
 import pandas as pd
 from django_tenants.utils import schema_context
 
 from api.common import log_json
 from api.models import Provider
+from masu.processor.aws.aws_report_parquet_processor import AWSReportParquetProcessor as trino_schema
```

Comment:
This alias seems like a misnomer. We should just use the object name for clarity.

Maybe a better approach is to move the schema data into separate frozen dataclasses. Then the `ReportParquetProcessor` could be composed of the relevant schema class, or the schema class could be used directly without the parquet processor bits.